velox/dwio/dwrf/writer/Writer.cpp

```diff
@@ -74,8 +74,14 @@ Writer::Writer(
                                     *options.encryptionSpec,
                                     options.encrypterFactory.get())
                               : nullptr);
-  writerBase_->initContext(options.config, pool, std::move(handler));
-
+  writerBase_->initContext(
+      options.config,
+      pool,
+      options.sessionTimezone,
+      options.adjustTimestampToTimezone,
+      std::move(handler));
+  common::testutil::TestValue::adjust(
+      "facebook::velox::dwrf::Writer::Writer", writerBase_.get());
```

Comment:
Where is this used?  If unused we should remove it.

Reply:
This is used when testing writing Timestamp data. I will delete it first.

Reply:
Already removed.